import.html

```diff
@@ -259,14 +247,21 @@ <h1 class="hero-heading">Add Your Project</h1>
                         <br/><select id="tileServer">
                             <option value="bing">bing</option>
                             <option value="digital_globe">digital_globe</option>
-                            <option value="google">google</option>
+                            <!-- <option value="google">google</option> -->
+							<option value="custom">custom</option>
```

Comment:
Another slight nitpick about indentation here. It's usually best to have them line up.